packages/core/src/extensions/Blocks/nodes/Block.ts

```diff
@@ -46,38 +43,31 @@ declare module "@tiptap/core" {
 export const Block = Node.create<IBlock>({
   name: "block",
   group: "block",
+  // A block always contains content, and optionally a blockGroup which contains nested blocks
+  content: "blockContent blockGroup?",
+  // Ensures content-specific keyboard handlers trigger first.
+  priority: 50,
+  defining: true,
+
   addOptions() {
     return {
       HTMLAttributes: {},
     };
   },
 
-  // A block always contains content, and optionally a blockGroup which contains nested blocks
-  content: "content blockgroup?",
-
-  defining: true,
-
   addAttributes() {
     return {
-      listType: {
-        default: undefined,
-      },
       blockColor: {
         default: undefined,
       },
       blockStyle: {
         default: undefined,
       },
-      headingType: {
-        default: undefined,
-        keepOnSplit: false,
-      },
     };
   },
 
   parseHTML() {
```

Comment:
maybe remove this? Let's do it after tests work, and also create a test for pasting a <span> or other unsupported tag

Reply:
Since copy/pasting from other sites doesn't work very well after the restructuring, I think we should just leave it as is for now and try to fix the issue later, since it's quite a bit more complex than originally expected.